adam-core/src/main/scala/org/bdgenomics/adam/models/ReferenceRegion.scala

```diff
@@ -85,6 +102,23 @@ case class ReferenceRegionWithOrientation(region: ReferenceRegion,
   def end: Long = region.end
 }
 
+object ReferenceRegionContext {
+  implicit val referenceRegionOrdering = ReferenceRegionOrdering
+  implicit val referenceRegionWithOrientationOrdering = ReferenceRegionWithOrientationOrdering
```

Comment:
Oh. Good question. I'm not sure... _Howaaayver_, I think that this is necessary to provide an implicit conversion from `ReferenceRegion` to `Ordering[ReferenceRegion]`, which is otherwise provided (???) by having `ReferenceRegion` extend `Ordered`. TBH, IDK.